pkgs/os-specific/linux/systemd/cryptsetup-generator.nix

```diff
@@ -13,6 +13,18 @@ stdenv.lib.overrideDerivation systemd (p: {
     ninja systemd-cryptsetup systemd-cryptsetup-generator
   '';
 
+  # The rpath to the shared systemd library is not added by meson. The
```

Comment:
This comment needs to be updated. Meson sets the rpath now, but it is not done in this derivation because ninja `install` target (which takes care of that) is not used.

Reply:
Comment updated